src/SFA.DAS.ProviderApprenticeshipsService.Application/Commands/SubmitCommitment/SubmitCommitmentCommand.cs

```diff
@@ -10,6 +12,8 @@ public class SubmitCommitmentCommand : IAsyncRequest
 
         public string Message { get; set; }
 
-        public string SaveOrSend { get; set; }
+        public AgreementStatus AgreementStatus { get; set; }
```

Comment:
i don't think we should BUT all of the command handlers currently do this. instead the orchestrator should map to domain entities (ie. the commands should know about entities not api types)

i think this is a refactor we need to do at some point across all the commands as a separate ticket